cohortextractor/patients.py

```diff
@@ -1,27 +1,55 @@
-"""
-These methods don't *do* anything; they just return their name and arguments.
-This provides a friendlier API then having to build some big nested data
-structure by hand and means we can make use of autocomplete, docstrings etc to
-make it a bit more discoverable.
-"""
-
+""" This module provides the methods for making a study definition """
+# These methods don't *do* anything; they just return their name and arguments.
+# This provides a friendlier API than having to build some big nested data
+# structure by hand and means we can make use of autocomplete, docstrings etc to
+# make it a bit more discoverable.
```

Comment:
Ah, sorry, I wasn't very clear last time!

This whole block can be a docstring.  The content is great and by making it a docstring means tooling can pick it up in the future 😄